apps/src/music/views/Video.jsx

```diff
@@ -65,7 +65,7 @@ const Video = ({id, onClose}) => {
             height="100%"
             style={{border: 'none'}}
             src="https://www.youtube-nocookie.com/embed/ab2SBrfkKXU?rel=0"
-            title=""
+            title="Project Beats Introduction"
```

Comment:
I considered translating this, but confirmed that the embedded video retains the English title even when viewing youtube in another language. The title is also encoded into the first few frames of the video itself, so sticking to English seems reasonable here.